cmd/legacy/configread/configread.go

```diff
@@ -42,7 +43,7 @@ func Read(v *viper.Viper) (string, error) {
 		return "", err
 	}
 
-	value := strings.TrimSpace(v.GetString(params.ViperKey()))
+	value := strings.TrimSpace(vipertools.GetString(v, params.ViperKey()))
```

Comment:
We don't want to strip quotes from config values, only from command line params.